sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_models.py

```diff
@@ -134,6 +136,41 @@ def _to_generated(self):
         return CallLocator(kind=self.kind,
                            group_call_id=self.id)
 
+class ChannelAffinity(object):
+    """Channel affinity for a participant.
+
+    All required parameters must be populated in order to send to Azure.
+
+    :ivar channel: Channel number to which bitstream from a particular participant will be written.
```

Comment:
nit: in docstring, channel comes first but in actual signature, target_participant comes first in order

Reply:
reordered